ble_controller/doc/CHANGELOG.rst

```diff
@@ -0,0 +1,62 @@
+.. _ble_controller_changelog:
+
+Changelog
+#########
+
+All notable changes to this project are documented in this file.
+
+ble_controller 0.2.0-1.prealpha
+*******************************
+
+Updated BLE controller with bugfixes and updated APIs.
+
+Added
+=====
+
+* Added API for fetching build revision information
+* Added ``ble_controller_rand_vector_get_blocking()``
```

Comment:
I have done this now, except in the CHANGELOG.rst. The reason is that the changelog should be immutable and function (and file) names referred to in it may at some point be deprecated and that would break the linkage.